source/unified-test-format/unified-test-format.md

```diff
@@ -534,6 +534,14 @@ The structure of this object is as follows:
         client.
     - `serverApi`: Optional [serverApi](#serverapi) object.
 
+    <span id="entity_client_autoEncryptOpts"></span>
+
+    - `autoEncryptOpts`: Optional object with the following fields:
+        - `kmsProviders`: The same as in [`clientEncryption`](#entity_clientEncryption).
+        - `keyVaultNamespace`: Optional, a namespace to the key vault collection. Defaults to "keyvault.datakeys".
+        - `bypassAutoEncryption`: Optional, a boolean to indicate whether or not auto encryption should be bypassed.
+            Defaults to `false`.
```

Comment:
At the cost of adding scope: I recommend adding other `AutoEncryptionOpts` fields in this PR: `schemaMap`, `extraOptions`, `encryptedFieldsMap`, `bypassQueryAnalysis`, and `keyExpirationMS`.

DRIVERS-2295 is planned next. I expect the other `AutoEncryptionOpts` will be needed to migrate over the legacy tests. Adding these options in this PR might avoid an additional schema version.

I think `tlsOptions` can be left out (not currently used by legacy spec tests, and the representation is not specified).

Reply:
I originally assumed this PR was for [DRIVERS-2295](https://jira.mongodb.org/browse/DRIVERS-2295), but just realized it doesn't have an associated JIRA issue.

I concur with adding everything that will be necessary to port of the legacy CSFLE tests. And there should absolutely be a JIRA isssue attached to this PR so we can communicate driver changes for the new schema version.